src/exec.js

```diff
@@ -21,6 +21,7 @@ function execSync(cmd, opts, pipe) {
   }
 
   var tempDir = _tempDir();
+  var paramsFile = path.resolve(tempDir + '/' + common.randomFileName());
```

Comment:
This doesn't need to be fixed now, but I think the paths should be joined with `path.join`, instead of just concatenating the strings... future improvements.